tests/test_task.py

```diff
@@ -115,16 +112,16 @@ def test_derived_item(derived_item_task: Task) -> None:
     items = derived_item_task.process(**derived_item_task.parameters)
     links = [lk for lk in items[0]["links"] if lk["rel"] == "derived_from"]
```

Comment:
maybe some checks here that `items` is exactly a list of length 1?

Reply:
That'd be a separate issue, this PR just for linting and formatting changes.

Reply:
ah, i didn't realize the `next` changes was a lining fix